firedrake/preconditioners/bddc.py

```diff
@@ -52,13 +54,19 @@ def initialize(self, pc):
         bddcpc.setOperators(*pc.getOperators())
         bddcpc.setType(PETSc.PC.Type.BDDC)
 
+        # Allow viewing preconditioning matrix for debugging purposes
+        P.viewFromOptions('-view_pmat', bddcpc)
```

Comment:
Should this still be here?

Reply:
That would lengthen the string, I would either keep as is or entirely remove it.